FV3/atmos_cubed_sphere/driver/fvGFS/atmosphere.F90

```diff
@@ -139,6 +139,7 @@ module atmosphere_mod
 
 #include <fms_platform.h>
 !$ser verbatim use mpi
+!$ser verbatim USE m_serialize, ONLY: fs_is_serialization_on
```

Comment:
Line gave me a slight pause because of the usage of `USE` vs. the above `use` and I don't know whether it makes a difference.

Reply:
oh good point, it doesn't make a functional difference, but I should make them consistent, thanks for pointing that out!